modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase

```diff
@@ -913,6 +913,125 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12600. As a buyer, I can verify the order items are split on the placed order when the 'Show Separate Order Items' toggle is enabled"
+	@priority = 5
+	test CanVerifyOrderItemsAreSplitOnThePlacedOrder {
+		property custom.properties = "company.security.strangers.verify=false${line.separator}jsonws.web.service.paths.excludes=";
+		property portal.acceptance = "true";
+
+		task ("Given a buyer user connected to the Commerce Account") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the admin enables the toggle 'Show Separate Order Items' for Minium Portal channel") {
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Show Separate Order Items",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
+		}
+
+		task ("And the buyer logs in and clicks 2 times on the 'Add to cart' button for MIN55861") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			for (var count : list "1,2") {
+				Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+				Click(
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+			}
```

Comment:
Can you add to the cart with different quantities? This way you can assert both rows later on during the test. Thanks!